source/interfaces/q2studio.rst

```diff
@@ -36,6 +36,7 @@ Next we need to install it (both as a Python package, and as a Node.js package):
 .. command-block::
    :no-exec:
 
+   conda install gevent nodejs
```

Comment:
This should probably have some explicit channel(s) listed (`-c defaults ...` etc), as well as `--override-channels` to ensure that a user's local conda config doesn't install the wrong thing.

See here for an example of this, in action:

https://github.com/qiime2/busywork/blob/644376876df798ef34f29c0494be596f9335f36e/ci/master/bin/test.sh#L10-L14

Reply:
I should say, the main reason I propose adding explicit channels in (and disabling the channels in their local conda config) is mostly just a support thing - it helps us ensure that they are installing the same versions of things (from the same sources), as us. Functionally its probably no different to omit the explicit channels.

Reply:
Sounds good to me!